gobblin-core/build.gradle

```diff
@@ -50,6 +50,8 @@ dependencies {
 
   runtime externalDependency.protobuf
 
+  runtime files('./src/main/resources/lib/ojdbc6.jar')
```

Comment:
I'm not quite sure how this will be looked by the installed Gobblin (extracted tarball).
Do you plan to pack this 3rd party dependency to gobblin-core?